draft-ietf-protected-audience-key-value-services.md

```diff
@@ -537,42 +537,42 @@ serialized to string.
 This algorithm describes how the Key Value Service MAY generate a response to a request.
 
 The input is a list of [deterministically encoded CBOR](https://www.rfc-editor.org/rfc/rfc8949.html#name-deterministically-encoded-c) `partitionOutputs` in {{response-schema}} as well as
```

Comment:
You may want to specify how the Key Value Service creates the "partitionOutputs"
 - as the output from the UDF given the keys (and what other fields) in the corresponding input
 - describe constraints on UDF?